FlowCrypt/Controllers/Msg/MessageViewController.swift

```diff
@@ -0,0 +1,442 @@
+//
```

Comment:
Maybe the this VC was brought back in by mistake?

Reply:
Yes, it's reference was deleted from the project, but file was still stored in repository, removed it.